app/models/job_proxy_dispatcher.rb

```diff
@@ -192,25 +197,23 @@ def self.waiting?
     Job.where(:state => 'waiting_to_start').exists?
   end
 
-  def pending_jobs(target_class = VmOrTemplate)
-    class_name = target_class.base_class.name
+  def pending_jobs(job_class = VmScan)
     @zone = MiqServer.my_zone
-    Job.order(:id)
-       .where(:state           => "waiting_to_start")
-       .where(:dispatch_status => "pending")
-       .where(:target_class    => class_name)
-       .where("zone is null or zone = ?", @zone)
-       .where("sync_key is NULL or
-         sync_key not in (
-           select sync_key from jobs where
-             dispatch_status = 'active' and
-             state != 'finished' and
-             (zone is null or zone = ?) and
-             sync_key is not NULL)", @zone)
+    job_class.order(:id)
```

Comment:
@hsong-rh Please review the change here to use `VmScan` by default to look up `Job` objects instead of using `Job.where(:target_class => "VmOrTemplate")`.  Looks good to me but I want to confirm with you.